engine/gui/src/gui.cpp

```diff
@@ -4226,6 +4233,8 @@ namespace dmGui
 
         InternalNode* n = GetNode(scene, node);
         out_n->m_Node = n->m_Node;
+        out_n->m_Node.m_HasResetPoint = false;
+        out_n->m_Node.m_ResetPointProperties = 0;
```

Comment:
According to documentation `guidelines.reset_nodes()`shouldn't affect dynamic nodes:
![CleanShot 2025-03-11 at 08 03 00@2x](https://github.com/user-attachments/assets/b506fa42-2a6e-452e-9ce6-fee771e127ed)

That means we don't need this data here